### DISCUSSION TITLE
How to use a type in TypeScript?

### DISCUSSION THREAD

**Discussion by wsw70** [moment/luxon#1092] [Q&A]
I would like to add a type to my TypeScript code : 

    import { DateTime } from 'luxon'

    const now: →the type of DateTime← = DateTime.now() 

I installed [@types/luxon](https://www.npmjs.com/package/@types/luxon) but I do not know what to put in `→the type of DateTime←`

**Comment by Andrew-Marks-Trisept** [comment]
Should just be "DateTime".  I'm doing stuff like this:

`const luxonDate: DateTime = DateTime.fromISO(value)`
